k8/images/geopandas/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# syntax = docker/dockerfile:1.1.7-experimental
+ARG FROM
+FROM ${FROM}
+
+WORKDIR /usr/src/app
+
+RUN cat /etc/os-release
+RUN apt-get install libgeos-dev
+
+# Must build from source to be able to run tests
+# https://github.com/geopandas/geopandas
+RUN git clone \
+    --branch v0.8.1 \
+    --depth 1 \
+    https://github.com/geopandas/geopandas \
+    .
+
+RUN --mount=type=cache,target=/root/.cache/pip \
+    pip install \
+    cython \
+    fiona \
+    pyproj \
+    geos \
+    pytest \
+    pytest-cov \
+    pytest-xdist \
+    fsspec \
+    SQLalchemy \
+    pyarrow \
+    geopy \
+    mapclassify>=2.2.0 \
+    pandas \
+    matplotlib \
+    shapely \
+    pygeos \
```

Comment:
You can leave out pygeos here, I think. It's an experimental replacement for Shapely, and will be upstreamed into Shapely, so no need to install it for the purpose of this docker image